Documentation/In-depth/TheInstallTool/Index.rst

```diff
@@ -175,7 +191,6 @@ This is the recommended way.
 
 **Test Mail Setup** allows you to test the mail setup of TYPO3 by sending a
 test mail.
-
```

Comment:
This line should be kept.

Reply:
This line has not been rmoved
It is stil there:

> It offers you the possibility to create a file :file:`.htaccess` in the TYPO3 root directory.
> This is the recommended way.

Reply:
Please have a closer look at the line I commented on. You seem to have dropped an empty line which should be reverted.

Reply:
I did not notice that.